### PR TITLE
src: release Node.js version 8.10.0-cr

### PR DESCRIPTION
**Pull request by danbev**
This commit updates the node to 8.10.0 candidate release version.

I'm was not sure if this PR should be opened against the `8.x` branch or not. Feel free to close and open against the correct (perhaps a new branch) or let me know and I'll do that.

### CODE REVIEW

**Comment by lance** [comment]
@danbev not sure what would be best for this. Maybe a new branch `8.next`? I don't want a candidate release to be what you get if you pull `8.x` from docker hub

**Comment by danbev** [comment]
@lance A branch named like that sounds good to me. I'll need to update this with the environment variable for node-gyp (that is why I marked it as in progress). Might not get around to that until tomorrow. I an create a branch and reopen this PR against it.

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ ENV NPM_RUN=start \
     NPM_CONFIG_LOGLEVEL=info \
     NPM_CONFIG_PREFIX=$HOME/.npm-global \
     PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
+    npm_config_tarball=/usr/share/node/node-v${NODE_VERSION}-headers.tar.gz \
```
Comment (lance) [inline Dockerfile]: Minor nit - very minor! But it seems odd to me to have a variable name in all lowercase when all of the others are in all caps. The inconsistency is a little odd to me.

Comment (danbev) [reply Dockerfile]: Not sure what I was thinking there, let me fix that. Thx

**Pull request merged**